flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/sdk/Extensions.kt

```diff
@@ -0,0 +1,125 @@
+package flank.scripts.shell.firebase.sdk
+
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.onError
+import flank.scripts.github.objects.GitHubCreateIssueRequest
+import flank.scripts.github.objects.GitHubCreateIssueResponse
+import flank.scripts.github.objects.GitHubUpdateIssueRequest
+import flank.scripts.github.patchIssue
+import flank.scripts.github.postNewIssue
+import flank.scripts.zenhub.FLANK_REPO_ID
+import flank.scripts.zenhub.convertIssueToEpic
+import flank.scripts.zenhub.objects.ConvertToEpicRequest
+import flank.scripts.zenhub.objects.Issue
+import flank.scripts.zenhub.objects.UpdateEpicRequest
+import flank.scripts.zenhub.updateEpic
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.coroutineScope
+import java.lang.Exception
+import kotlin.system.exitProcess
+
+internal suspend fun SDKUpdateContext.createEpicIssue() = coroutineScope {
+    val (updates, subIssues) = createSubIssues()
+
+    println("** Create new epic")
+    val epic = createEpic(updates)
+
+    convertIssueToEpic(
+        zenhubToken = zenhubToken,
+        issueNumber = epic,
+        payload = ConvertToEpicRequest(subIssues)
+    )
+}
+
+internal suspend fun SDKUpdateContext.updateOpenedEpic() = coroutineScope {
+    val (updates, subIssues) = createSubIssues()
+    requireNotNull(openedIssue)
+
+    println("** Update existing epic")
+    patchIssue(
+        githubToken = githubToken,
+        issueNumber = openedIssue.number,
+        payload = GitHubUpdateIssueRequest(
+            title = "Implement new gcloud features [$newVersion]",
+            body = openedIssue.body + "\n" + updates
```

Comment:
move it to const val and put it on `:common` module please

Reply:
Hm...I think I don't fully understand. Both `openedIssue.body` and `updates` are variables.

Reply:
I mean new line `\n`😄

Reply:
🤦  arghh